resolwe_bio/processes/alignment/star.yml

```diff
@@ -12,9 +12,9 @@
         image: resolwebio/legacy:1.0.0
     resources:
       memory: 32768
-      cores: 1
+      cores: 10
```

Comment:
At some point we should also try using more cores in _STAR genome index_ process. If RAM usage is an issue we have the parameter `genomeChrBinNbits` exposed which heavily influences RAM usage.

Reply:
Agree. But for now, the guideline i was given was 10 cores max.